MetaMask/metamask-extension
# Updating all caret icons for dark mode 🥕

Explanation: Updating all caret icons to use newly create caret icon components.
- Removed old svg images
- Created components for all caret icons up, down, left, right
- Created stories for any components that were updated for easy visual review

Manual testing steps:  
  - Go to latest build of storybook and go through the list of updated components using search bar
  - Look at only the caret icons
  - Check dark mode by clicking the 🌙  icon in the toolbar of storybook
  
List of updated components:
- `Icons`
- `AddNetwork`
- `ConfirmPageContainerHeader` (controls need to be like the below)
<img width="765" alt="Screen Shot 2022-03-08 at 8 20 33 AM" src="https://user-images.githubusercontent.com/8112138/157279947-c9a482e8-a12b-46a9-a449-65a4e07dcba7.png">

- `NetworkDisplay`
- `TabBar` (needs to be mobile view)
<img width="1024" alt="Screen Shot 2022-03-08 at 8 23 50 AM" src="https://user-images.githubusercontent.com/8112138/157280624-a950ede1-bbf4-4289-938c-0f3aa0b85648.png">

- `Dropdown`
- `SenderToRecipient` (needs to be `CARDS_VARIANT`)
<img width="1059" alt="Screen Shot 2022-03-08 at 8 25 09 AM" src="https://user-images.githubusercontent.com/8112138/157280847-a1cf8790-5f94-45a2-aa13-17bf9e01f462.png">


- `AddRecpient` NOT IN STORY LOOK AT SCREENSHOT BELOW with "Back to all" text
<img src="https://user-images.githubusercontent.com/8112138/157131677-0308d3c5-0d79-4021-8cc2-a82f55507fb9.png">

- `NetworksListItem`
- `SettingsSearchList` NO STORY
<img width="362" alt="Screen Shot 2022-03-08 at 9 22 28 AM" src="https://user-images.githubusercontent.com/8112138/157292216-1d507576-f3b8-4434-ba86-3278d5681d9d.png">
<img width="964" alt="Screen Shot 2022-03-08 at 9 22 37 AM" src="https://user-images.githubusercontent.com/8112138/157292220-96bb4c44-2a5a-492a-8f0e-fe6f347da760.png">


## Screenshots
<img width="1440" alt="Screen Shot 2022-03-06 at 6 55 58 PM" src="https://user-images.githubusercontent.com/8112138/157131662-d6b1da96-1592-4d91-8305-da7a7b636266.png">
<img width="1440" alt="Screen Shot 2022-03-06 at 6 56 04 PM" src="https://user-images.githubusercontent.com/8112138/157131665-f3bced4c-ae08-4d16-843d-e69ffd69cb9f.png">
<img width="1440" alt="Screen Shot 2022-03-06 at 9 36 07 AM" src="https://user-images.githubusercontent.com/8112138/157131667-c86dbcb5-c160-42fa-bc7c-0cf5264441c4.png">
<img width="1440" alt="Screen Shot 2022-03-06 at 9 36 29 AM" src="https://user-images.githubusercontent.com/8112138/157131668-115ca2ef-8229-4ec0-8f08-4ec40b27ecd7.png">
<img width="1440" alt="Screen Shot 2022-03-06 at 9 37 01 AM" src="https://user-images.githubusercontent.com/8112138/157131671-6826d6d6-4f79-4cd3-95e5-0567d9a8949a.png">
<img width="1440" alt="Screen Shot 2022-03-06 at 9 55 53 AM" src="https://user-images.githubusercontent.com/8112138/157131674-cb383548-5244-4782-b5e2-ece31151eca6.png">
<img width="1440" alt="Screen Shot 2022-03-06 at 9 57 00 AM" src="https://user-images.githubusercontent.com/8112138/157131675-9c9f4b3c-64a8-4aaf-981a-ce2abbe1711e.png">
<img width="1440" alt="Screen Shot 2022-03-06 at 11 06 05 AM" src="https://user-images.githubusercontent.com/8112138/157131677-0308d3c5-0d79-4021-8cc2-a82f55507fb9.png">
<img width="1436" alt="Screen Shot 2022-03-06 at 11 06 40 AM" src="https://user-images.githubusercontent.com/8112138/157131678-35357bdd-8c13-4116-8dcd-84ef0f6382ba.png">
<img width="348" alt="Screen Shot 2022-03-07 at 1 48 37 PM" src="https://user-images.githubusercontent.com/8112138/157131681-3dc8000a-7e70-4748-b7b5-19da2a197a01.png">
<img width="340" alt="Screen Shot 2022-03-07 at 1 48 40 PM" src="https://user-images.githubusercontent.com/8112138/157131683-66ae33b4-2f54-436a-bf8a-80be8dcd1209.png">
<img width="262" alt="Screen Shot 2022-03-07 at 1 59 55 PM" src="https://user-images.githubusercontent.com/8112138/157131684-02697fb4-1de3-4792-8ae6-673e893fb643.png">
<img width="236" alt="Screen Shot 2022-03-07 at 1 59 59 PM" src="https://user-images.githubusercontent.com/8112138/157131686-3ed77ce6-4caa-4728-b154-ec56029aa5c3.png">
<img width="255" alt="Screen Shot 2022-03-07 at 2 09 02 PM" src="https://user-images.githubusercontent.com/8112138/157131689-81aea0f4-be4d-46b8-bd47-738fac174853.png">
<img width="215" alt="Screen Shot 2022-03-07 at 2 09 06 PM" src="https://user-images.githubusercontent.com/8112138/157131692-81cf2193-374f-4f1c-9627-e456e357b223.png">
<img width="1233" alt="Screen Shot 2022-03-07 at 2 52 46 PM" src="https://user-images.githubusercontent.com/8112138/157131694-05f23cc2-b2ad-4c4f-be08-8d7833f3fd86.png">
<img width="1440" alt="Screen Shot 2022-03-07 at 2 53 01 PM" src="https://user-images.githubusercontent.com/8112138/157131697-5fe39e44-dc85-4179-b178-87d67df2a6e8.png">
